packages/testcontainers/src/wait-strategies/host-port-wait-strategy.test.ts

```diff
@@ -20,7 +20,7 @@ describe("HostPortWaitStrategy", { timeout: 180_000 }, () => {
         .withExposedPorts(8081)
         .withStartupTimeout(0)
         .start()
-    ).rejects.toThrowError(/Port \d+ not bound after 0ms/);
+    ).rejects.toThrowError(/Port (?:\d+|\d+\/\w+) not bound after 0ms/);
```

Comment:
There are no cases where the protocol won't be included

```suggestion
    ).rejects.toThrowError(/Port \d+\/(tcp|udp) not bound after 0ms/);
```